src/main/scala/com/imaginea/activegrid/core/models/AppSettingsNeo4jWrapper.scala

```diff
@@ -1,112 +1,137 @@
-    if (settingsType.equalsIgnoreCase("AUTH_SETTINGS"))
-      updateOrDeleteSettings(settingsToDelete, labels("HAS").toString, "DELETE")
-    else
-      updateOrDeleteSettings(settingsToDelete, labels("HGS").toString, "DELETE")
-  }
-
-  def getSettingsByRelation(rootNode: Node, relationName: String): Map[String, String] = {
-    val relationNode = getRelationNodeByName(rootNode, relationName)
-    relationNode match {
-      case Some(node) => node.getAllProperties.mapValues(_.toString()).toMap[String, String]
-      case None => Map.empty[String, String]
-    }
-  }
-
-  def getRelationNodeByName(rootNode: Node, relationName: String): Option[Node] = {
-    rootNode.getRelationships.find(relation => relation.getType.name() == relationName) match {
-      case Some(relationship) => Some(relationship.getEndNode)
-      case None => None
-    }
-  }
-
-  def updateOrDeleteSettings(settings: Map[String, String], relationName: String, updateOrDelete: String): Future[ExecutionStatus] = Future {
-    rep.withTx {
-      neo => {
-        rep.withTx { neo =>
-          val settingNodes = rep.getAllNodesWithLabel(labels("AS").toString)(neo).headOption
-          settingNodes match {
-            case Some(rootNode) => val relationNode = getRelationNodeByName(rootNode, relationName)
-              relationNode match {
-                case Some(dbnode) =>
-                  val todelete = if (updateOrDelete.equalsIgnoreCase("DELETE")) true else false
-                  settings.foreach {
-                    case (k, v) =>
-                      if (todelete)
-                        dbnode.removeProperty(k)
-                      else
-                        dbnode.setProperty(k, v.toString)
-                  }
-                case None => ExecutionStatus(false)
-              }
-            case None => ExecutionStatus(false)
-          }
-        }
-      }
-    }
-    ExecutionStatus(true)
-  }
-
-  def setNodeProperties(n: Node, settings: Map[String, String]) {
-    settings.foreach {
-      case (k, v) => n.setProperty(k, v);
-    }
-  }
-}
-
+/*
+ * Copyright (c) 1999-2013 Pramati Technologies Pvt Ltd. All Rights Reserved.
+ *
+ * This software is the confidential and proprietary information of Pramati Technologies.
+ * You shall not disclose such Confidential Information and shall use it only in accordance with
+ * the terms of the source code license agreement you entered into with Pramati Technologies.
+ */
+package com.imaginea.activegrid.core.models
+
+import com.typesafe.scalalogging.Logger
+import org.neo4j.graphdb.Node
+import org.slf4j.LoggerFactory
+
+import scala.collection.JavaConversions._
+import scala.collection.immutable.HashMap
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
+/**
+  * Created by sivag on 6/10/16.
+  */
+object AppSettingsNeo4jWrapper {
+
+  val labels: HashMap[String, String] =

```

Comment:
Formated as per the below reference 
http://docs.scala-lang.org/style/indentation.html